zerver/openapi/zulip.yaml

```diff
@@ -19231,6 +19425,61 @@ components:
 
             New in Zulip 8.0 (feature level 191). Previously, groups
             could be mentioned if and only if they were not system groups.
+    Invite:
+      type: object
+      description: |
+        A dictionary containing details about an invitation.
+      additionalProperties: false
+      properties:
+        id:
+          type: integer
+          description: |
+            The unique ID of the invitation link object.
+        invited_by_user_id:
+          type: integer
+          description: |
+            The [user ID](/api/get-user) of the user who created the invitation.
+
+            **Changes**: New in Zulip 3.0 (feature level 22). Previously, `ref`
+            field was used which contained the Zulip display email address of the
+            user who created the invitation.
```

Comment:
I'd revise this to be something like:

"... level 22), replacing the `ref` field, which contained the Zulip display ..."